test/api.js

```diff
@@ -19,6 +19,21 @@ describe('client-side api', function () {
 				.call(done);
 		});
 
+		it('should ensure that duplicate bundleName-messageName pairs are ignored', done => {
+			e.apis.extension.listenFor('testMessageName', 'testBundleName', done);
+			e.apis.extension.listenFor('testMessageName', 'testBundleName', done);
```

Comment:
It does write an error (to the `Winston` error stream) but doesn't actually `throw` anything. So I ignore the error (couldn't figure out how to listen for error messages via Winston, or replace Winston with a mocked version), and just make sure that only one copy of the `testMessageName` listener gets registered.

Happy to remove this particular check if you think the code shouldn't work that way, I was just looking for some long-hanging fruit in the coverage reports to work on my unit testing knowledge in Node :)